parser/src/__tests__/invoke.test.ts

```diff
@@ -0,0 +1,41 @@
+import { parseMachinesFromFile } from "../parseMachinesFromFile";
+import { hashedId } from "../utils";
+
+describe("Invoke", () => {
+  it("Should allow for JSON stringifying anonymous invocations", () => {
+    const id = hashedId("() => {}");
+    const result = parseMachinesFromFile(`
+      createMachine({
+        invoke: {
+          src: "${id}",
```

Comment:
is this testing a correct thing though? this src here is not "anonymous" - it just has a very not friendly name 😉

Reply:
Yes, because the name MUST be deterministic. We're basically testing if hashedId ends up in the computed config.